rotkehlchen/api/server.py

```diff
@@ -268,6 +269,7 @@
     ('/database/info', DatabaseInfoResource),
     ('/database/backups', DatabaseBackupsResource),
     ('/locations/associated', AssociatedLocations),
+    ('/staking/', StakingResource),
```

Comment:
I am not sure having a generic staking resource is a good idea. We already have staking information for say ETH2. Also AdEx. Why are they not here? You could just for now do `/staking/kraken` and if at some point we add more that can be generic enough we can also use the `/staking` endpoint alone.

Reply:
I went with staking only because this endpoint takes an argument `location` and _should_ work for every staking protocol that has infomration in the `history_events` table. If you prefer I can change it to be `/staking/kraken` only